flink-runtime/src/main/java/org/apache/flink/runtime/io/network/buffer/LocalBufferPool.java

```diff
@@ -130,6 +130,10 @@ class LocalBufferPool implements BufferPool {
     @GuardedBy("availableMemorySegments")
     private final AvailabilityHelper availabilityHelper = new AvailabilityHelper();
 
+    /**
+     * Indicates this {@link LocalBufferPool} will request buffer from global pool when it becomes
+     * available.
+     */
```

Comment:
```suggestion
    /**
     * Indicates whether this {@link LocalBufferPool} has requested to be notified on the next time that global pool becoming available, so it can then request buffer from the global pool.
     */
```

Reply:
I'd suggest to also rename `requestingWhenAvailable` to `requestingNotificationOfGlobalPoolAvailable`.